core/db/migrate/20150515211137_fix_adjustment_order_id.rb

```diff
@@ -0,0 +1,66 @@
+class FixAdjustmentOrderId < ActiveRecord::Migration
+  def change
+    say 'Populate order_id from adjustable_id where appropriate'
+    execute(<<-SQL.squish)
```

Comment:
Do we still need this part even with b8b5e55661e2560437fb3b322ce284283e243dd8 and 30ae28f2e24001214737130835a50d8f9fb7cdca?